modules/search/pages/create-quick-index.adoc

```diff
@@ -0,0 +1,56 @@
+= Create a Search Index in Quick Mode
+:page-topic-type: guide
+:description: Use Quick Mode to create a Search index in Couchbase Capella with streamlined options.
+
+[abstract]
+{description}
+
+Use xref:create-search-index-ui.adoc[Advanced Mode] to add additional settings and more advanced options for your Search index and xref:customize-index.adoc#type-mappings[type mappings].
```

Comment:
Would be even more people to help people understand what exactly they're giving up when they choose quick mode or when it's the right choice.....(Choose Quick Mode when....)